lib/vdsm/tool/__init__.py

```diff
@@ -20,9 +20,39 @@
 from __future__ import absolute_import
 from __future__ import division
 import functools
+import logging
 import os
 
 
+LOGGER_NAME = "vdsm-tool"
+
+
+def setup_logging(log_file, verbosity, append):
+    level = logging.DEBUG if verbosity else logging.INFO
+    root_logger = logging.getLogger(LOGGER_NAME)
+    root_logger.setLevel(level)
+
+    if root_logger.handlers:
```

Comment:
Ignoring log_file?

Reply:
This is a save-guard in case it gets configured twice, not to add extra handlers.

Reply:
I see, OK.  It looked like a case when an external configuration is used (e.g. in other places in Vdsm), but it's not the case here.